src/MarketPriceWidget.java

```diff
@@ -0,0 +1,70 @@
+import javax.swing.*;
+import java.awt.*;
+
+/**
+ * MarketPriceWidgets consist of an image to identify an item, a text label with the item's name, and a modifiable
+ * text field displaying the item's value, in silver.
+ */
+public class MarketPriceWidget extends JPanel {
+
+    private final GridBagConstraints imageConstraints;
+    private final GridBagConstraints labelConstraints;
+    private final GridBagConstraints textBoxConstraints;
+
+    public MarketPriceWidget(Icon icon, String name, Long value) {
+        super(new GridBagLayout());
+        setBackground(Color.DARK_GRAY);
+        setSize(180, 60);
```

Comment:
Setting the size of the widget is redundant when the layout manager will handle it anyways.